hack/bump-common-instancetypes.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Copyright 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+target_branch=${1:-"main"}
+
+_common_instancetypes_base_url="https://github.com/kubevirt/common-instancetypes/releases/download"
+_cluster_instancetypes_path="data/common-instancetypes-bundle/common-clusterinstancetypes-bundle.yaml"
+_cluster_preferences_path="data/common-instancetypes-bundle/common-clusterpreferences-bundle.yaml"
+
+function latest_version() {
+    if [[ $target_branch == "main" ]]; then
+        curl --fail -s "https://api.github.com/repos/kubevirt/common-instancetypes/releases/latest" |
```

Comment:
Drop use of `--fail`, likely has no effect here?

Reply:
https://github.com/kubevirt/kubevirt/blob/0ca03aa05ccab5fefc6bfe7b474e5e05e18929a4/hack/bump-common-instancetypes.sh#L8-L14 - That's copied from your script in `kubevirt/kubevirt` and I think it does make sense here for the API calls.

Reply:
Ok, let's keep it then